util/SerializeUniverse.cpp

```diff
@@ -151,8 +151,13 @@ void Universe::serialize(Archive& ar, const unsigned int version)
         }
     }
 
-    ar  & BOOST_SERIALIZATION_NVP(m_stat_records);
-    DebugLogger() << "Universe::serialize : " << serializing_label << " " << m_stat_records.size() << " types of statistic";
+    if (Archive::is_saving::value && (!m_stat_records_publish)) {
```

Comment:
will this also stop stat records from begin saved in a save game file? that might be unexpected behaviour...

Reply:
Is `Universe::serialize` used in save game as well? How can I test if I'm in the serialization to file or to network?

Reply:
Yes.

I think checking if `GetUniverse().EncodingEmpire() == ALL_EMPIRES` would work. It's used in various serialization code to check if info is being sent to a particular empire / player.

Reply:
Fixed,